ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -5349,28 +5348,8 @@ let with_license_server_changes printer rpc session_id params hosts f =
             ~value:license_server
         )
         current_license_servers ;
-      let alerts =
-        Client.Message.get_since ~rpc ~session_id
-          ~since:(Date.of_unix_time (now -. 1.))
-      in
-      let print_if_checkout_error (ref, msg) =
-        if
-          false
-          || msg.API.message_name = fst Api_messages.v6_rejected
-          || msg.API.message_name = fst Api_messages.v6_comm_error
-          || msg.API.message_name
-             = fst Api_messages.v6_license_server_version_obsolete
-        then (
-          Client.Message.destroy ~rpc ~session_id ~self:ref ;
-          printer (Cli_printer.PStderr (msg.API.message_body ^ "\n"))
-        )
-      in
-      if alerts = [] then
-        raise e
-      else (
-        List.iter print_if_checkout_error alerts ;
-        raise (ExitWithError 1)
-      )
```

Comment:
Is this logic to `destroy` particular messages no longer necessary? Or is it performed by other tools now?